apps/odk_publish/tables.py

```diff
@@ -21,6 +21,14 @@ class FormTemplateTable(tables.Table):
         verbose_name="",
         attrs={"a": {"class": "text-primary-600 hover:underline"}},
     )
+    edit = tables.LinkColumn(
+        "odk_publish:edit-form-template",
+        args=[tables.A("project_id"), tables.A("pk")],
+        text="Edit",
+        orderable=False,
+        verbose_name="",
+        attrs={"a": {"class": "text-primary-600 hover:underline"}},
+    )
```

Comment:
I suggest removing both View and Edit columns and just making the title clickable:

```suggestion
    title_base = tables.LinkColumn(
        "odk_publish:form-template-detail",
        args=[tables.A("project_id"), tables.A("pk")],
        attrs={"a": {"class": "text-primary-600 hover:underline"}},
    )
```

Then add an edit button on the detail page:

```html
                <a href="{% url 'odk_publish:edit-form-template' request.odk_project.id form_template.id %}"
                   class="btn btn-outline ml-4">
                    Edit Form Template
                </a>
```